Code/Framework/AzGameFramework/AzGameFramework/Application/GameApplication.cpp

```diff
@@ -82,7 +82,7 @@ namespace AzGameFramework
 
         // Used the lowercase the platform name since the bootstrap.game.<config>.<platform>.setreg is being loaded
```

Comment:
This comment needs to be updated to remove platform, the filename is now just bootstrap.game.<config>.setreg